config/default.json

```diff
@@ -1,6 +1,7 @@
 {
   "mode": "bundled",
   "port": 8081,
+  "anchorControllerEnabled": false,
```

Comment:
`node-config-ts` uses the `default.json` to generate `Config.d.ts`. I guess `false` value is OK for now. The developer can always change it to `true` since it's just for developing purposes.